docs/execution-providers/DirectML-ExecutionProvider.md

```diff
@@ -136,4 +136,4 @@ A complete sample of onnxruntime using the DirectML execution provider can be fo
 * [ONNX models](https://docs.microsoft.com/en-us/windows/ai/windows-ml/get-onnx-model)
 * [DirectML Github](https://github.com/microsoft/DirectML)
 
-<p><a href="#">Back to top</a></p>
\ No newline at end of file
+<p><a href="#">Back to top</a></p>
```

Comment:
Um, I can't tell why this line is showing as a diff 🙃. I didn't touch it 🤔...

Reply:
It looks exactly the same!